### PR TITLE
Add AppImage section to supported platforms

### PR DESCRIPTION
**Pull request by michaelstingl**
Description added by @mmattel 

* Small installation documentation revision
* Adding AppImage
* Fix for #125 

Backport to 2.9 only

### CODE REVIEW

**Review by NannaBarz — APPROVED** [review]

**Comment by mmattel** [comment]
The original changes made needed some improvements to better fit into the context. Therefore I made a quick revision of the document and placed the additions to the correct location.

Backport to 2.9 only

**Comment by mmattel** [comment]
Just seen that issue #125 (Document additional installation dependency for nautilus extension on CentOS) can be added to that fix

**Comment by mmattel** [comment]
@fmoc mind to have a look to lines 29 to 41 with regards to your issue #125 if this is correct as written?

File: modules/ROOT/pages/installing.adoc [hunk 1]
```diff
@@ -2,33 +2,61 @@
...
+:desktop-clients-url: https://owncloud.com/desktop-app/
+:appimage-wikipedia-url: https://en.wikipedia.org/wiki/AppImage
+:gnome-keyring-url: https://wiki.gnome.org/Projects/GnomeKeyring/
+:kwalletmanager-url: https://utils.kde.org/projects/kwalletmanager/
+:ubuntu-lts-url: https://wiki.ubuntu.com/LTS
 
 == Introduction
 
-You can download the latest version of the ownCloud Desktop Synchronization Client from the https://owncloud.com/download/#desktop-clients[ownCloud download page]. There are clients for _Linux_, _macOS_, and _Microsoft Windows_.
+You can download the latest version of the ownCloud Desktop Synchronization Client from the {desktop-clients-url}[Download Desktop App]. There are clients for _Linux_, _macOS_, and _Microsoft Windows_ available.
```
Comment (phil-davis) [inline modules/ROOT/pages/installing.adoc:13]:
```suggestion
You can download the latest version of the ownCloud Desktop Synchronization Client from the {desktop-clients-url}[Desktop App Download] page. There are clients for _Linux_, _macOS_, and _Microsoft Windows_ available.
```

File: modules/ROOT/pages/installing.adoc [hunk 2]
```diff
@@ -2,33 +2,61 @@
...
+Full installation::
+Linux users must follow the instructions on the download page to add the appropriate repository for their Linux distribution, install the signing key, and then use their package managers to install the desktop sync client. Linux users will also update their sync clients via package manager, and the client will display a notification when an update is available.
++
+Linux users must also have a password manager enabled, such as {gnome-keyring-url}[GNOME Keyring] or {kwalletmanager-url}[KWallet], so that the sync client can login automatically.
++
+You will also find links to source code archives and older versions on the download page.
 
-NOTE: For Linux distributions, we support, if technically feasible, the latest 2 versions per platform and the previous Ubuntu https://wiki.ubuntu.com/LTS[LTS].
+AppImage::
+Starting with Desktop version 2.9, an {appimage-wikipedia-url}[AppImage] build of the ownCloud desktop sync client is available to support more Linux platforms. You can download the AppImage right at the {desktop-clients-url}[Linux section of the Download Desktop App].
```
Comment (phil-davis) [inline modules/ROOT/pages/installing.adoc:59]:
```suggestion
Starting with Desktop version 2.9, an {appimage-wikipedia-url}[AppImage] build of the ownCloud desktop sync client is available to support more Linux platforms. You can download the AppImage right at the {desktop-clients-url}[Linux section of the Download Desktop App] page.
```

File: modules/ROOT/pages/installing.adoc [hunk 3]
```diff
@@ -2,33 +2,61 @@
...
+** Note the Python extensions for nautilus (GNOME) and its forks nemo (Cinnamon) and caja (MATE) are available only in the third-party EPEL archives which needs to be installed separately before the ownCloud package can be installed. To do so, run
++
+[source,console]
+----
+sudo yum install epel-release
+----
 * Debian 10 & 11 (x86-64)
 * Fedora 33 & 34 & 35 (x86-64)
-* Ubuntu 20.04 & 21.04 & 21.10 (x86-64)
+** Same as with CentOS, the EPEL package needs to be installed before installing the ownCloud Desktop Client.
```
Comment (fmoc) [inline modules/ROOT/pages/installing.adoc:39]: You do not need EPEL on Fedora for the nautilus bindings. The packages are available on Fedora 33+.

Comment (mmattel) [reply modules/ROOT/pages/installing.adoc]: fixed and squashed

**Pull request merged**